### ISSUE TITLE
Bug: Incorrect dimension for concatenating encoded latents in data preprocessing

### ISSUE THREAD

**Issue by King-Wood-Shen** [CUC-MIPG/UniVid#2]
### Description
In the data preprocessing stage (`examples/wanvideo/train_univid.py`), video latents from 4 groups are concatenated along the **channel dimension (dim=1)** instead of the **temporal dimension (dim=2)**, causing incorrect tensor shapes.

### Current Behavior (Incorrect)
**File**: `examples/wanvideo/train_univid.py`, Line ~215
```python
encoded_latents = []
cnt = 0
for i in range(4):
    latents = self.pipe.encode_video(video[:, :, cnt:cnt+count_frames[i], :, :], **self.tiler_kwargs)[0]
    cnt += count_frames[i]
    encoded_latents.append(latents)

latents = torch.cat(encoded_latents, dim=1)  # ❌ Wrong: concatenates on channel dim
```

**Result**: 
- Input per group: `[1, 16, 5, 60, 104]` (for 17 frames → 5 latent frames)
- After concatenation: `[1, 64, 5, 60, 104]` ❌
- Channels incorrectly expand from 16 to 64

### Expected Behavior (Correct)
Should concatenate along **temporal dimension (dim=2)**:
```python
latents = torch.cat(encoded_latents, dim=2)  # ✓ Correct: temporal concatenation
```

**Expected Result**:
- After concatenation: `[1, 16, 20, 60, 104]` ✓
- Preserves 16 channels, extends temporal dimension

### Evidence from Inference Code
The inference code confirms temporal concatenation is expected:

**File**: `diffsynth/pipelines/univid_wan_video.py`, Line ~248
```python
for i in range(4):
    frame = self.decode_video(latents[:, :, vae_f:vae_f+num_frames[i], :, :], **tiler_kwargs)
    #                                  ^^^^^^^^^^^^^^^^^^^^^^^^^^^^
    #                                  Slicing on dim=2 (temporal)
    vae_f += num_frames[i]
```

Decoding slices on `dim=2`, proving training should also concatenate on `dim=2`.

### Impact
1. **Shape mismatch**: DiT expects `[B, 16, T, H, W]` but receives `[B, 64, T', H, W]`
2. **Training failure**: Model cannot load preprocessed `.tensors.pth` files correctly
3. **Inconsistency**: Preprocessing and inference use different dimension conventions

### Reproduction Steps
1. Run data preprocessing:
```bash
   python examples/wanvideo/train_univid.py --task data_process \
     --dataset_path data/train/test --num_frames 180 --n_clipA 17 --n_clipB 17
```
2. Load saved tensor and check shape:
```python
   data = torch.load("path.tensors.pth")
   print(data["latents"].shape)  # Shows [1, 64, 5, 60, 104] instead of [1, 16, 20, 60, 104]
```

### Proposed Fix
```diff
# examples/wanvideo/train_univid.py, line ~215

  encoded_latents = []
  cnt = 0
  for i in range(4):
      latents = self.pipe.encode_video(video[:, :, cnt:cnt+count_frames[i], :, :], **self.tiler_kwargs)[0]
      cnt += count_frames[i]
      encoded_latents.append(latents)
- latents = torch.cat(encoded_latents, dim=1)
+ latents = torch.cat(encoded_latents, dim=2)
```

### Additional Notes
This appears to be a typo that went unnoticed. The training code's slicing logic also uses `dim=2`:
```python
# train_univid.py, line ~460
noisy_latents[:, :, sum(count_frames[:3]):, :, :] = ...
#                   ^^^^^^^^^^^^^^^^^^^^^^^^ temporal slicing
```

Would appreciate confirmation and will submit a PR if needed. Thank you!
